content/en/docs/contributing/external-issuers.md

```diff
@@ -36,6 +36,15 @@ corresponding issuer resource exists in Kubernetes. If these are both true, it
 will then use the information inside that issuer resource to attempt to create a
 signed certificate, based upon the certificate request.
 
+Issuers must also ensure that before signing, the `CertificateRequest` is
+[`Approved`](../../concepts/certificaterequest/#approval). If the
+`CertificateRequest` is not `Approved`, the issuer should not process it.
+Issuers are not responsible for approving `CertificateRequests`. Issuer
+implementations may choose to support older versions of cert-manager (pre
+`v1.3`) by signing `CertificateRequests` that will never have an approval
+condition set.  Disabling this gate check should be an optional flag, and the
+gate **enabled by default**.
```

Comment:
So perhaps we should update the sample-external-issuer to demonstrate how we'd implement this gate.